deploy/helm/charts/templates/csi-driver.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.csiDriver.create -}}
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: cstor.csi.openebs.io
+spec:
+  # Supports persistent inline volumes.
+  volumeLifecycleModes:
+    - Persistent
```

Comment:
Ephemral mode needs be added , check here https://github.com/openebs/charts/blob/1ef07f4e6719f9eb923aa0ec5abdd03e146d41aa/2.3.0/cstor-operator.yaml#L596